assets/css/app.less

```diff
@@ -2,2444 +2,16 @@
 # We need this frontmatter for Jekyll to compile the less code
 ---
 
-//
 // Import variables
-// ==================================================
-// ==================================================
-
 @import "bootstrap/less/bootstrap.less";
 @import "user10/variables";
 @import "app-variable-overrides";
 @import "user10/utilities";
 @import "vertical-carousel";
 @import "lity";
 
+// Import global styles
+@import "global";
 
-
```

Comment:
What's the advantage of moving this from `app.less` to `global.less`?

Reply:
I wasn't able to get the code to reference a file other than `app.less`, so I kept `app.less` as the main point of entry and then had it point to all the "global" settings in `global.less`. This set the stage for having less styles in `global.less` and more in page-specific LESS files.